app/helpers/fhir_helper.rb

```diff
@@ -161,39 +176,54 @@ def patient_from_fhir(patient, default_jurisdiction_id)
   end
 
   # Build a FHIR US Core Race Extension given Sara Alert race booleans.
-  def to_us_core_race(white, black_or_african_american, american_indian_or_alaska_native, asian, native_hawaiian_or_other_pacific_islander)
+  def to_us_core_race(races)
     # Don't return an extension if all race categories are false or nil
-    return nil unless [white, black_or_african_american, american_indian_or_alaska_native, asian, native_hawaiian_or_other_pacific_islander].include?(true)
+    return nil unless races.values.include?(true)
 
     # Build out extension based on what race categories are true
     FHIR::Extension.new(url: 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-race', extension: [
-      white ? FHIR::Extension.new(
+      races[:white] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2106-3', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'White')
       ) : nil,
-      black_or_african_american ? FHIR::Extension.new(
+      races[:black_or_african_american] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2054-5', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Black or African American')
       ) : nil,
-      american_indian_or_alaska_native ? FHIR::Extension.new(
+      races[:american_indian_or_alaska_native] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '1002-5', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'American Indian or Alaska Native')
       ) : nil,
-      asian ? FHIR::Extension.new(
+      races[:asian] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2028-9', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Asian')
       ) : nil,
-      native_hawaiian_or_other_pacific_islander ? FHIR::Extension.new(
+      races[:native_hawaiian_or_other_pacific_islander] ? FHIR::Extension.new(
         url: 'ombCategory',
         valueCoding: FHIR::Coding.new(code: '2076-8', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Native Hawaiian or Other Pacific Islander')
       ) : nil,
+      races[:race_other] ? FHIR::Extension.new(
+        url: 'text',
+        valueString: 'other'
+      ) : nil,
```

Comment:
Realizing my original suggestion to represent "Other" this way won't work, because the `extension:text` within the US Core Race extension is intended to describe all of the Patient's race concept's, and there can only be one (which we have below in lines 219-226). My new suggestion is to use `extention:detailed`, using the "Other Race" code from the [detailed race value set](https://www.hl7.org/fhir/us/core/ValueSet-detailed-race.html). I was trying to avoid this before since that code is deprecated within HL7, but I see no other way.

```suggestion
      races[:race_other] ? FHIR::Extension.new(
        url: 'detailed',
        valueCoding: 'FHIR::Coding.new(code: '2131-1', system: 'urn:oid:2.16.840.1.113883.6.238', display: 'Other Race')'
      ) : nil,
```

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/767b8765f1830cb0e34e8bc8b1adbbe48c0786f5#diff-402d451d1f7019322100522f6898c3153444edb8ddb963e181a457c84a2ca6b6R209